### DISCUSSION TITLE
Activate scene from scheduled job

### DISCUSSION THREAD

**Discussion by fpytloun** [zwave-js/zwave-js-ui#4062] [Q&A] [answered]
Hello, I would like to have job that will attempt to turn off heating every few hours to have a fallback mechanism directly on controller in case Homeassistant is down or cannot use ZWave properly (if it is functioning properly, it will just turn on heating again).

I created scene like this that works as expected but I cannot find a script to use as a scheduled job. Can someone please give me example or some clues?

<img width="926" alt="image" src="https://github.com/user-attachments/assets/c2254b55-3db2-4445-90e7-ef70cd002760" />

**Comment by kpine** [comment]
Scenes are executed using MQTT. https://zwave-js.github.io/zwave-js-ui/#/guide/mqtt?id=execute-scene

**Reply by fpytloun** [reply]
Yes, that I saw. But it also requires some external system to be running and accessible. I want failsafe mechanism that will work even when network is unreachable. There must be some way how to execute from script, just cannot find it 🙁

**Reply by kpine** [reply, marked as answer]
You can try a driver function and see if it will let you call [`this.zwaveClient._activateScene`](https://github.com/zwave-js/zwave-js-ui/blob/6b262178a7ad7a24585fe56af2c37af6524e1a1a/api/lib/ZwaveClient.ts#L2706-L2721).

https://zwave-js.github.io/zwave-js-ui/#/usage/driver_function

**Reply by fpytloun** [reply]
Yes, this works! Thank you.

**Reply by robertsLando** [reply]
@fpytloun FYI you could also opt for broadcast/multicast command in this case as the valueId you are controlling are the same ones (and that's also much faster)
